hindi/actor4/ompuri.md

```diff
@@ -0,0 +1,2 @@
+ओम पुरी हिन्दी फिल्मों के एक प्रसिद्ध अभिनेता हैं। इनका जन्म १८ अक्टूबर 1950 में अम्बाला नगर में हुआ। इन्होने ब्रिटिश तथा अमेरिकी सिनेमा में भी योगदान किया है। ये पद्मश्री पुरस्कार विजेता भी हैं, जोकि भारत के नागरिक पुरस्कारों के पदानुक्रम में चौथा पुरस्कार है।
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.